### PR TITLE
completed bonus

### PR DESCRIPTION
**Pull request by jamesshi567**
Addition of A3 and Lab2 answers to VIRUSxDISVRY repo for bonus

### CODE REVIEW

File: virus/img/rdrp.jpg
Comment (ababaian) [inline virus/img/rdrp.jpg:1]: Please move your image files to a folder named after you virus as shown in the instructions.

File: virus/virTemplate.md
Comment (ababaian) [inline virus/virTemplate.md:1]: Please rename the file to your `<virus_name>.md` and adjust the image folders above likewise (fixing hte links in the markdown).

Please do not delete/modify the `virTemplate.md` file

File: virus/virTemplate.md [hunk 1]
```diff
@@ -1,40 +1,81 @@
-# {Q1A Paper Title}
-written by: [yourname](https://github.com/yourname *optional link*)
+# Possible novel rhabdovirus whose anomalous structural genes will g-"rhabdo" your attention
+written by: [James Shi](https://github.com/jamesshi567)
 
-### {Q1B Virus Name}
-*{Q1B, etiology of virus name}*
+### Apiovirus anomala
+*We believe based on current data that this virus has the closest relation to the Apis rhabdovirus. However, it remains unclassified beyond the family leve, as many rhabdoviruses are due to their high diversity. Thus, there is inherently some uncertainty in the assignment of a binomial name for this virus. I assigned a genus mimicking how "Apis" refers to bees, a highly suspected host for this virus. The second term anomala refers to the unexpected genomic organization of this virus.*
```
Comment (ababaian) [inline virus/virTemplate.md]: include query coverage and percentage identity between these two sequences in the opening paragraph where making this comparison.

**Comment by jamesshi567** [comment]
Modified the files according to the comments. Git bash was being weird so I had to resort to the newbie method of uploading some of the files. Let me know if the issues are resolved. Thanks!

**Comment by ababaian** [comment]
There's a conflic ton the branch because you deleted `virTemplate.md` here, restore that file in the PR and it should work.

**Comment by jamesshi567** [comment]
Hello Artem, so sorry about the delay. I was demolished by a difficult BCH final and then caught up studying for this one and forgot to check for updates from your end. It shows now that there's no more conflicts with the branch, so I believe I have fixed the issue. Do let me know if I missed something.

**Comment by ababaian** [comment]
No worries, good luck tonight!

**Pull request merged**